src/lib.rs

```diff
@@ -1353,6 +1353,23 @@ where
     }
 }
 
+impl<K, V> Clone for FlurryHashMap<K, V, RandomState>
+where
+    K: Sync + Send + Clone + Hash + Eq,
+    V: Sync + Send + Clone,
+{
+    fn clone(&self) -> Self {
+        let cloned_map = FlurryHashMap::with_capacity(self.count.load(Ordering::SeqCst));
```

Comment:
You should be able to use `self.len()` now that #6 has landed!

Reply:
Awesome, I was looking for it and couldn't find it. It is much better, thanks!